Package.swift

```diff
@@ -0,0 +1,42 @@
+// swift-tools-version:5.0.1
+
+//
+//  Copyright 2020 Square Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import PackageDescription
+
+let package = Package(
+	name: "Stagehand",
+	platforms: [
+		.iOS(.v10),
+	],
+	products: [
+		.library(
+			name: "Stagehand",
+			targets: ["Stagehand"]
+		),
+	],
+	targets: [
+		.target(
+			name: "Stagehand",
+			dependencies: [],
+			path: "Stagehand/Classes/Core"
```

Comment:
I’d recommend re-arranging your directory structure to match SPM’s expectations. But this can always be done later.

Reply:
Updated the directory structure. I think this is how SPM recommends.